src/main/java/in/projecteka/gateway/common/model/GatewayResponse.java

```diff
@@ -10,5 +10,6 @@
 @Getter
 @AllArgsConstructor
 public class GatewayResponse {
```

Comment:
should implement serializable?

Reply:
Since it's part of Throwable class, sonar suggested make it serialisable. Except one single property which is of type `object` inside `ErrorResult` I made them serialisable as sonar suggested.